src/HomeWork_2/data-access/insert_list_of_users.ts

```diff
@@ -0,0 +1,10 @@
+import { Client } from 'pg';
+import { User } from '../types';
+
+export const insertListOfUsers = (client: Client, users: User[]) => {
+  users.forEach(({ id, login, password, age, is_deleted }) => {
+    client.query('INSERT INTO Users (id, login, password, age, is_deleted) VALUES ($1, $2, $3, $4, $5)', [id, login, password, age, is_deleted])
```

Comment:
FYI, it should be possible to do this via `sequelize` model, so that there is no need to write any SQL

Reply:
Got it, but I guess it was a first part of task to use just bare access to DB